### ISSUE TITLE
ZED Mini camera on Nvidia AGX Orin not detected.

### ISSUE THREAD

**Issue by segapss** [stereolabs/zed-sdk#588]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues.
- [X] This issue is not a question, feature request, or anything other than a bug report directly related to this project.

### Description

Hey,
I have installed JetPack 5.1.1 on Nvidia AGX Orin for project using Stereo ZED Mini camera. But when I have connected it on Orin, it could not recognise device. When I put `lsusb` on terminal, it shows no device title apart from ID and BUS number. But weird thing is, when I connect device on host laptop, it is recognised easily! I have also tried all direction of TypeC cable, but result were all same. [overlay_usb_35.3.1.tbz2](https://developer.nvidia.com/embedded/jetson-linux-r3531)  And on here, I even have installed 'overlay_usb_35.3.1.tbz2' option in case of UART receiving data issue, but result is same as I mentioned above. https://forums.developer.nvidia.com/t/zed-camera-not-detected-on-power-up/175908 This guy seems to have a similar issue with me, but I want to get more clear and exact solution for recognition of ZED Mini camera on Nvidia AGX Orin. Because installation of ZED SDK on Orin would be reasonable only after the successful recognition of ZED camera on Orin. Thanks.

### Steps to Reproduce

1. Connect ZED Mini camera on Nvidia AGX Orin
2. Type `lsusb` for device recognition.

### Expected Result

Recognition of ZED Mini camera on Orin. For example(tested on host laptop),
`Bus 001 Device 010: ID 2b03:f681 STEREOLABS ZED-M Hid Device`

### Actual Result

No recognition on Orin. For example, 
`Bus 001 Device 010: ID 2b03:f681`

### ZED Camera model

ZED Mini

### Environment

```shell
Nvidia AGX Orin Developer Kit 32GB
Jetpack 5.1.1(L4T 35.3.1)
Ubuntu 20.04
No ZED SDK yet. Because I need a test before installation.

Host pc: Ubuntu 20.04
No ZED SDK also on here.
```

**Comment by Myzhar** [comment]
Already handled by the support team
